### PR TITLE
tying moose events to file event emitter

### PR DESCRIPTION
**Pull request by matszczygiel**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * Add file descriptors resolver callback based on content URI - Unix platform only
 * Fix database foreign keys not being enabled
 * Remove `ServiceStop` error
+* Disallow file cancels from the sending side
```
Comment (Lipt0nas) [inline changelog.md:11]: Is this right?

Comment (matszczygiel) [reply changelog.md:11]: It's not a feature of this PR but wanted to sneak that in.
It's true, automated retires do not allow canceling files from the sender, we discussed it yesterday on Slack

Comment (Lipt0nas) [reply changelog.md:11]: ah okay, wasn't sure since it wasn't related to the PR 👍

**Review by Lipt0nas — APPROVED** [review]
A very nice cleanup, less moose code that litters random places 👍

**Pull request merged**